src/client/websocket/WebSocketShard.js

```diff
@@ -128,24 +125,28 @@ class WebSocketShard extends EventEmitter {
      * @type {?NodeJS.Timer}
      * @private
      */
-    Object.defineProperty(this, 'helloTimeout', { value: null, writable: true });
+    Object.defineProperty(this, 'helloTimeout', { value: undefined, writable: true });
 
     /**
      * If the manager attached its event handlers on the shard
      * @type {boolean}
      * @private
      */
     Object.defineProperty(this, 'eventsAttached', { value: false, writable: true });
-  }
 
-  /**
-   * Average heartbeat ping of the websocket, obtained by averaging the WebSocketShard#pings property
-   * @type {number}
-   * @readonly
-   */
-  get ping() {
-    const sum = this.pings.reduce((a, b) => a + b, 0);
-    return sum / this.pings.length;
+    /**
+     * A set of guild IDs this shard expects to receive
+     * @type {Set<string>}
```

Comment:
This property is initially set as `undefined`, per line 142
```suggestion
     * @type {?Set<string>}
```